aisdc/attacks/worst_case_attack.py

```diff
@@ -540,13 +546,20 @@ def make_report(self) -> dict:
             "dummy_attack_experiments_logger"
         ] = self._get_dummy_attack_metrics_experiments_instances()
 
+        output_pdf = {}
+        output_pdf["attack_metrics"] = self.attack_metrics
+        output_pdf[
+            "dummy_attack_metrics"
+        ] = self._unpack_dummy_attack_metrics_experiments_instances()
+        output_pdf["metadata"] = self.metadata
+
         if self.args.report_name is not None:
             json_report = report.create_json_report(output)
             with open(f"{self.args.report_name}.json", "w", encoding="utf-8") as f:
                 f.write(json_report)
 
-        #    pdf = report.create_mia_report(output)
-        #    pdf.output(f"{self.args.report_name}.pdf", "F")
+            pdf = report.create_mia_report(output_pdf)
```

Comment:
probably a legacy from me, but `pdf` is also a bad filename :-). `pdf_output`?

Reply:
@simonrnss I'm reviewing your review now apparently! I disagree with `pdf_output` as a name - since the dictionary is called `output_pdf` - those names are too similar and could definitely get confusing!

`pdf_report` instead?